Formula/grafana.rb

```diff
@@ -0,0 +1,123 @@
+require "language/go"
+require "language/node"
+require "open3"
+
+class Grafana < Formula
+  desc "Gorgeous metric visualizations and dashboards for timeseries databases."
+  homepage "http://grafana.org"
+  url "https://github.com/grafana/grafana/archive/v3.0.4.tar.gz"
+  sha256 "f26a374326e64a8f83c57fdf916ba1c8524dd55002dfe628b4854d05fed3715a"
+
+  head "https://github.com/grafana/grafana.git"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  def install
+    ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/npm/bin"
+    ENV["GOPATH"] = buildpath
+    grafana_path = buildpath/"src/github.com/grafana/grafana"
+    grafana_path.install Dir["*"]
+    grafana_path.install ".jscs.json", ".jsfmtrc", ".jshintrc", ".bowerrc"
+
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd grafana_path do
+      # Might do it differently for head vs. release
+      system %q(sed -i.bak 's/"sass-lint": "^1.6.0",/"sass-lint": "1.7.0",/;' package.json)
```

Comment:
This should be using an `inreplace`, but if we can drop `head` and drop this I'm game for that.